fix(curriculum): Divided step 13 into 2 steps; updated step 13 and added step 14 (#54472)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6624e0ec0bf47017eed84b1f.md

---
id: 6624e0ec0bf47017eed84b1f
title: Step 14
challengeType: 20
dashedName: step-14
---

# --description--

The first kind of cipher you are going to build is called a *Caesar* cipher.  Specifically, you will take each letter in your message, find its position in the alphabet, take the letter located after 3 positions in the alphabet, and replace the original letter with the new letter.

To implement this, you will use the `.find()` method discussed in the previous step. Modify your existing `.find()` call passing it `text[0]` as the argument instead of `'z'`.

# --hints--

> You should call the `.find()` method on the alphabet variable and pass `text[0]` to the method.

Reviewer: ```suggestion
You should modify your existing `alphabet.find('z')` call passing `text[0]` to the method.
```

```js

> assert.match(code, /^alphabet\.find\s*\(\s*text\s*\[\s*\s*0\s*\]\s*\)/m)

Reviewer: ```suggestion
assert.match(code, /^alphabet\.find\s*\(\s*text\s*\[\s*0\s*\]\s*\)/m)
```
